prompts.md

```
@@ -0,0 +1,20 @@
+
+Prompt 1:
+
+Tengo un repositorio en github con una tarea que debo realizar, debo descargarlo, o te comparto el link de los archivos?
+
+
+
+
+
+Prompt 2:
+
+https://github.com/LIDR-academy/AI4Devs-intro-202506 Esta es la url del Repositorio. Debo realizar la tarea que consiste en crear una pagina web usando javascript que reciba una cadena de texto y la devuelva al contrario, es decir, si introduzco hola, debe responder aloh. Esto usando los archivos index.html y scripts.js dentro de la carpeta templates. Allí también hay un ejemplo en imagen de como debe quedar la pagina web (res/reversestring-example.png). Falta información o así esta completo para hacer la pagina?
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)